python/Produce_Config_File.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import xlrd
+from array import array
+import numpy as np
+from ROOT import TGraph
+from ROOT import TMultiGraph
+import ROOT as r
+import argparse
+from optparse import OptionParser
+from options import parser
+import sys
+
+#parser = argparse.ArgumentParser(add_help=True)
+#parser.add_argument('file', nargs='+')
+#parser.add_argument('--test',action="store",dest="test",help="Select the type of test: QC3, QC4_HV QC4_SS")
+if len(sys.argv)==1:
+    parser.print_help()
+    sys.exit(1)
+
+
+
+
```

Comment:
Typically we try to limit unnecessary white space in coding as it makes it harder to see all the code at once (making it harder to understand).  At most you should have a single empty line between blocks of code.